wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,321 @@
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2012-05-01 | 2012-11-30 |  |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2012-11-30 | *2013-01-16* |  |
+
+## 2013
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | *2012-11-30* | 2013-01-16 |  |
+| ![][flag_DE] [ShadowSoul](https://osu.ppy.sh/users/494970) | 2013-01-16 | 2013-01-21 |  |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2013-01-21 | 2013-10-04 |  |
+| ![][flag_FI] [thelewa](https://osu.ppy.sh/users/475021) | 2013-10-04 | 2013-10-05 |  |
+| ![][flag_DE] [ShadowSoul](https://osu.ppy.sh/users/494970) | 2013-10-05 | 2013-10-20 |  |
+| ![][flag_KR] [Reimu-Desu](https://osu.ppy.sh/users/948713) | 2013-10-20 | 2013-12-29 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2013-12-29 | *2014-01-27* |  |
+
+## 2014
+
+*Note: ppv2 was introduced on 27th January 2014.*
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | *2013-12-29* | 2014-01-27 |  |
+| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 2014-01-27 | 2014-06-20 |  |
+| ![][flag_KR] [sayonara-bye](https://osu.ppy.sh/users/713266) | 2014-06-20 | 2014-08-17 |  |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2014-08-17 | *2015-02-06* |  |
+
+## 2015
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | *2014-08-17* | 2015-02-06 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2015-02-06 | 2015-03-25 |  |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2015-03-25 | 2015-11-22 |  |
+| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2015-11-22 | 2015-11-28 |  |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2015-11-28 | 2015-12-17 |  |
+| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2015-12-17 | 2015-12-19 |  |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2015-12-19 | *2016-05-12* |  |
+
+## 2016
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | *2015-12-19* | 2016-05-12 |  |
+| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2016-05-12 | 2016-06-25 |  |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-06-25 | 2016-06-28 |  |
+| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2016-06-28 | 2016-10-04 |  |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-10-04 | 2016-10-05 |  |
+| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 2016-10-05 | 2016-10-10 |  |
+| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 2016-10-10 | 2016-10-18 |  |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-10-18 | 2016-10-21 |  |
```

Comment:
I am not entirely sure if this is a continuous reign. If you look at the leaderboard on [2016-10-18](https://web.archive.org/web/20161018110255/https://osu.ppy.sh/p/pp) and [2016-10-20](https://web.archive.org/web/20161020182026/https://osu.ppy.sh/p/pp) the PP changes from:

1. Cookiezi 12,831pp -> 12,837pp
2. Angelsim 12,801pp -> 12,836pp

Due to the data gap on 2016-10-19, it's possible that Angelsim had a 1-day reign. Simultaneously https://www.reddit.com/r/osugame/comments/69fdad/players_who_hit_1/ suggests that Angelsim had 12,836pp on 2016-10-18.

I can't find anything that would clarify this, but it would be helpful if anyone does.

Reply:
I think it's technically possible but it's more unlikely than not, as looking into that entire time period I couldn't find anything that suggests that Angelsim had a 1-day reign on that specific date.